apps/admin-x-activitypub/src/api/activitypub.ts

```diff
@@ -53,24 +53,6 @@ export class ActivityPubAPI {
         return [];
     }
 
-    get outboxApiUrl() {
-        return new URL(`.ghost/activitypub/outbox/${this.handle}`, this.apiUrl);
-    }
-
-    async getOutbox(): Promise<Activity[]> {
```

Comment:
We no longer need this as we can pass `includeOwn` when making a request to the `activities` endpoint